src/main/java/life/qbic/io/commandline/PostmanCommandLineOptions.java

```diff
@@ -21,6 +21,11 @@
       description = "openBIS user name")
   public String user;
 
+  @Option(
+          names = {"-p", "--password:env"},
+          description = "provide name of environment Variable to read in password")
+  public String environmentVariableName;
```

Comment:
What do you think about including the purpose of the environment variable in its name?
Something like `passwordEnvVariable` could be an option.